release_tools/versions.sh

```diff
@@ -1,5 +1,5 @@
-version=1.3.0
-previous_version=1.3.0
+version=1.3.0_alpha1
+previous_version=1.2.17
```

Comment:
shouldn't this be 1.3.0_alpha1?

Reply:
@mpreisler No, it shouldn't. 1.3.0_alpha1 it has not been released yet. There is no `openscap-1.3.0_alpha1.tar.gz` tarball published on Github and there is no tag `1.3.0_alpha1` in git.

Reply:
why would we compare to 1.2.17? of course it won't be compatible... by design...

Reply:
@mpreisler Because 1.2.17 is the latest released version.

Reply:
well OK but comparing ABI to it makes zero sense

Reply:
@mpreisler Why?

Reply:
because of course it won't be compatible, it's a different branch

Reply:
@mpreisler In the report it is easily visible which symbols were removed and added.